src/manager/checkpoint/proof/v1.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+use crate::lotus::LotusClient;
+use anyhow::anyhow;
+use cid::Cid;
+use fvm_shared::clock::ChainEpoch;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct V1Proof {
+    tip_set: Vec<Cid>,
+    states: Vec<Cid>,
```

Comment:
Why is `states` a `Vec<Cid>`?

Reply:
Oh, I see. You set the parent root for all the blocks in the tipset. We only have one block per tipset so I would suggest using `Cid` instead of `Vec<Cid>` and assigning the one for block 0 (the only one present). I would probably do the same also for the `tip_set` (although this one is not as critical, we can leave the `Vec<Cid>`.

Reply:
updated